README.md

```diff
@@ -115,6 +147,8 @@ Unlike Pythonic Devnet, which supported the gateway and feeder gateway API, Devn
 
 The JSON-RPC API is reachable via `/rpc` and `/` (e.g. if spawning Devnet with default settings, these URLs have the equivalent functionality: `http://127.0.0.1:5050/rpc` and `http://127.0.0.1:5050/`)
 
+> Out of Starknet **trace** API RPC methods, only `starknet_simulateTransactions` is supported.
```

Comment:
Not sure if `>` is appropriate for this kind of note, it's usually used for [blockquotes](https://www.markdownguide.org/basic-syntax/#blockquotes-1)